embeddings/pipeline/lightning_sequence_labeling.py

```diff
@@ -45,8 +47,20 @@ def __init__(
         task_model_kwargs: Optional[Dict[str, Any]] = None,
         task_train_kwargs: Optional[Dict[str, Any]] = None,
     ):
+        output_path = Path(output_path)
+        task_train_kwargs = (
+            task_train_kwargs if task_train_kwargs else self.DEFAULT_TASK_TRAIN_KWARGS
+        )
+        task_model_kwargs = (
+            task_model_kwargs if task_model_kwargs else self.DEFAULT_TASK_MODEL_KWARGS
+        )
+        tokenizer_name = tokenizer_name if tokenizer_name else embedding_name
+        datamodule_kwargs = (
+            datamodule_kwargs if datamodule_kwargs else self.DEFAULT_DATAMODULE_KWARGS
+        )
```

Comment:
note that this is changed in #173 as these kwargs get updated now

Reply:
ok